### ISSUE TITLE
Missing Native Apple Silicon Version

### ISSUE THREAD

**Issue by conradwt** [xtuml/models#338]
The macOS build for native Apple Silicon isn't an available download option at this time.  Are there plans on the roadmap to start producing Apple SIlicon native builds?

**Comment by cortlandstarrett** [comment]
Yes.  And some experimentation has been done (was done last summer/fall timeframe).  At that time, there were still a few lingering issues with Eclipse and the supported JVMs.

Here is the tracking issue:  https://support.onefact.net/issues/12315

If you do some experimentation, please report your results here or in the above issue (12315).  IIRC, the editor worked fine.  It was integration with model compilers that still needed work.
